File: t/samples/hello_world/csharp/HelloWorld.cs
using System;

public class HelloWorld {
  private static int _id_seq = 0;
  private int _id;
  public static int hello = 1;

  public HelloWorld() {
    this._id = (HelloWorld._id_seq++);
  }

  public void say() {
    Console.WriteLine("Hello, world! My id is " + _id);
  }

  public void destroy() {
    Console.WriteLine("Goobdye, world! My id is " + _id);
  }

  private void private_method() {
    hello = 2;
    Console.WriteLine(hello);
  }
}

File: t/samples/mixed/hello_world.py
class HelloWorld:
  __id_seq = 0
  hello = 1 

  def __init__(self):
    self.__id = __id_seq + 1
    __id_seq += 1
  

  def say(self):
    print("Hello, world! My is id " + self.__id)
  

  def destroy(self):
    print("Goodbye, world! My id is " + self.__id)
  

  def __private_method():
    hello = 2
    print(hello)
